sharding-proxy/sharding-proxy-backend/src/main/java/org/apache/shardingsphere/shardingproxy/backend/text/TextProtocolBackendHandlerFactory.java

```diff
@@ -104,4 +111,15 @@ private static TextProtocolBackendHandler createDALBackendHandler(final DALState
         }
         return new UnicastBackendHandler(sql, backendConnection);
     }
+
+    private static String trimSqlComment(final String sql) {
+        String result = sql;
+        if (sql.startsWith(COMMENT_PREFIX)) {
+            result = sql.substring(sql.indexOf(COMMENT_SUFFIX) + 2).trim();
+        }
+        if (sql.endsWith(SQL_END)) {
```

Comment:
`;` will not be passed to proxy, please verify it again.

Reply:
![image](https://user-images.githubusercontent.com/30677017/65476390-5a7b7d00-deb5-11e9-82bf-a127c53ac33e.png)
I think it may be that the client is unable to recognize the statement, resulting in no processing.

Reply:
ok, it beyond client, let's retain this logic